services/frontend/.env

```diff
@@ -1,4 +1,8 @@
-REACT_APP_API_URL=https://jungle.eosio.cr
-REACT_APP_GRAPHQL_HTTP_URL=http://localhost:8088/v1alpha1/graphql
-REACT_APP_GRAPHQL_WS_URL=ws://localhost:8088/v1alpha1/graphql
-REACT_APP_EOS_API_URL=https://jungle.eosio.cr
\ No newline at end of file
+REACT_APP_GRAPHQL_HTTP_URL=https://graphql-jungle.eosrate.io/v1alpha1/graphql
+REACT_APP_API_URL=https://jungle.cryptolions.io
+REACT_APP_GRAPHQL_WS_URL=wss://graphql-jungle.eosrate.io/v1alpha1/graphql
+REACT_APP_EOS_API_URL=https://jungle.cryptolions.io
+REACT_APP_EOS_API_HOST=jungle3.cryptolions.io
+REACT_APP_EOS_API_PORT=443
+REACT_APP_EOS_API_PROTOCOL=https
+REACT_APP_EOS_CHAIN_ID=e70aaab8997e1dfce58fbfac80cbbb8fecec7b99cf982a9444273cbc64c41473
```

Comment:
@tetogomez  This is the chain ID for Jungle2

We need to set to jungle3 chain id in order to work with the specified endpoint `2a02a0053e5a8cf73a56ba0fda11e4d92e0238a4a2aa74fccf46d5a910746840`